modules/services/repositories/src/debug/kotlin/com/android/billingclient/api/ProductDetailsModule.kt

```diff
@@ -30,28 +30,28 @@ private class MockingProductDetailsInterceptor : ProductDetailsInterceptor {
             .build()
         private val products
             get() = listOf(
-                yearlyTestPlusProduct,
```

Comment:
Removed left over test product id after replacing ids in 
https://github.com/Automattic/pocket-casts-android/pull/2973 and added referral offer to the `yearlyPlusProduct`.